doc/releases/releases.yml

```diff
@@ -29,6 +29,8 @@ releases:
   reef:
     target_eol: 2025-08-01
     releases:
+      - version: 18.2.6
+        released: 2025-04-21
```

Comment:
Should this be the 16th instead of the 21st or are we not releasing until then?

Reply:
No this is when we paln to send an email

Reply:
Thank you for putting this at the top of the list in this file.  We've seen that this is important for proper generation of the the timeline.